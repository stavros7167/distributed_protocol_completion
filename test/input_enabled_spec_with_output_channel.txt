environment E {
  inputs []
  outputs [a, b]
  initial e0
  e0 a! e1
  e1 b! e0
  e1 a! e0
}

process P {
  states [p0]
  inputs [a, b]
  input_enabled [c]
  outputs [c]
  initial p0
}
