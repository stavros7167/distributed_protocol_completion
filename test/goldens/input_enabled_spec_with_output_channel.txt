Parsing file test/input_enabled_spec_with_output_channel.txt
In automaton P, message c used in input enabled spec but not is not an input of the automaton.
Error interpreting model.
